domains/transfer/Transfer.API/API.IntegrationTests/Transfer/Api/v2024_01_03/Controllers/TransferAgreementHistoryEntriesControllerTests.cs

```diff
@@ -7,16 +7,16 @@
 using API.IntegrationTests.Factories;
 using API.IntegrationTests.Shared;
 using API.Transfer.Api.Services;
-using API.Transfer.Api.v2023_11_23.Dto.Requests;
-using API.Transfer.Api.v2023_11_23.Dto.Responses;
+using API.Transfer.Api.v2024_01_03.Dto.Requests;
+using API.Transfer.Api.v2024_01_03.Dto.Responses;
 using FluentAssertions;
 using Newtonsoft.Json;
 using NSubstitute;
 using VerifyTests;
 using VerifyXunit;
 using Xunit;
 
-namespace API.IntegrationTests.Transfer.Api.v2023_11_23.Controllers;
+namespace API.IntegrationTests.Transfer.Api.v2024_01_03.Controllers;
```

Comment:
Same controller could support different versions, so version should be kept out of namespaces.

Reply:
Done:

https://github.com/Energinet-DataHub/energy-origin/pull/684/commits/84de4b074372535c545f2314f9930f0aa7050a88